canvas_modules/harness/test_resources/parameterDefs/readonly_paramDef.json

```diff
@@ -424,6 +424,15 @@
         ]
       }
     ],
+    "description": {
+      "default": "Readonly description",
+      "link": {
```

Comment:
The description wouldn't include the link details.  That comes from the help section if included.  

- If only description then there is no link and just text.
- If there is only help then it works like today and the icon opens the help page
- If there is both description and help then we'd have the description text with the link from the help section.

Reply:
Sure, I have updated PR as per above conditions.